stories/feed-list/feed-list.stories.js

```diff
@@ -0,0 +1,433 @@
+                title="John Doe"
+            ></div>
+                 
+            <div class="fd-feed-list__content">
+                <p class="fd-feed-list__text">
+                    <span class="fd-feed-list__name">Joe Doe: </span>
+                    <span>Maecenas convallis velit quis felis dictum, in ultrices quam faucibus. Morbi tempor eu elit eu consequat. Sed at lorem a ex consequat pharetra. Etiam convallis odio at vulputate venenatis. In sit amet pharetra urna. Ut nulla nisi, porta at ligula a, elementum ullamcorper lectus. Suspendisse blandit, risus nec vestibulum volutpat, nulla neque rhoncus dolor, vitae rutrum lectus neque nec ligula. Maecenas pulvinar, ligula nec fringilla volutpat, urna quam rutrum tellus, vi...
+                    <a class="fd-link fd-feed-list__more" tabindex="0">More</a>        
+                </p>
+                
+                <div class="fd-feed-list__footer">
+                    <small class="fd-feed-list__footer--date">Aug 20, 2020</small>
+                </div>
+            </div>
+        </div>
+    </div>
+`;
+
+showMoreButton.storyName = 'Show More Button';
+showMoreButton.parameters = {
+    docs: {
+        storyDescription: 'More Button'
+    }
+};
+
+export const showLessButton = () => `
+    <div class="fd-feed-list" aria-label="Feed List Item with show less text example">
+        <div class="fd-feed-list__body">
+            <div 
+                class="fd-avatar fd-avatar--s fd-avatar--thumbnail fd-feed-list__thumb" 
+                style="background-image: url(http://lorempixel.com/460/620/people/7/)" 
+                role="img" 
+                aria-label="John Doe"
+                title="John Doe"
+            ></div>
+                 
+            <div class="fd-feed-list__content">
+                <p class="fd-feed-list__text">
+                    <span class="fd-feed-list__name">Joe Doe: </span>
+                    <span>Maecenas convallis velit quis felis dictum, in ultrices quam faucibus. Morbi tempor eu elit eu consequat. Sed at lorem a ex consequat pharetra. Etiam convallis odio at vulputate venenatis. In sit amet pharetra urna. Ut nulla nisi, porta at ligula a, elementum ullamcorper lectus. Suspendisse blandit, risus nec vestibulum volutpat, nulla neque rhoncus dolor, vitae rutrum lectus neque nec ligula. Maecenas pulvinar, ligula nec fringilla volutpat, urna quam rutrum tellus, vitae elementum elit est malesuada nunc. Maecenas eu risus posuere, volutpat justo in, ultricies sem
+                    <a class="fd-link fd-feed-list__more" tabindex="0">LESS</a>        
+                </p>
+                
+                <div class="fd-feed-list__footer">
+                    <small class="fd-feed-list__footer--date">Aug 20, 2020</small>
+                </div>
+            </div>
+        </div>
+    </div>
+`;
+
+showLessButton.storyName = 'Show Less Button';
+showLessButton.parameters = {
+    docs: {
+        storyDescription: 'Less Button'

```

Comment:
Similarly, feed list items can display a _LESS_ link that will revert the text back to when it was truncated.